src/components/shared/Entity/ValidationErrorSummary.tsx

```diff
@@ -98,10 +99,15 @@ function ValidationErrorSummary() {
 
     return (
         <Collapse in={filteredErrorsList.length > 0} timeout="auto">
-            <Alert severity="error">
+            <Alert
+                severity="error"
+                variant={theme.palette.mode === 'dark' ? 'standard' : 'filled'}
```

Comment:
If this is needed on all alerts that we either need a wrapper of this needs to be set higher up.

Reply:
It is not needed on all alerts.